.env.example

```diff
@@ -45,3 +45,16 @@ AUTH0_TOKEN_URL=
 AUTH0_CLIENT_ID=
 AUTH0_CLIENT_SECRET=
 AUTH0_AUDIENCE=
+
+# email settings
+EMAIL_BACKEND='django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST='smtp.gmail.com'
+EMAIL_USE_TLS=True
+EMAIL_PORT=587
+DEFAULT_FROM_EMAIL=Plio <no-reply@plio.in>
+EMAIL_HOST_USER= # sender's email-id
+# password associated with above email-id
+# for accounts with 2-factor authentication, you have to create a application-specific
+# passwords. More on that here:
+# https://dev.to/abderrahmanemustapha/how-to-send-email-with-django-and-gmail-in-production-the-right-way-24ab
+EMAIL_HOST_PASSWORD=
```

Comment:
references

Reply:
do we want to put the backend waali cheez in env file?
For example, for authentication we had these 
```py
AUTHENTICATION_BACKENDS = (
    "social_core.backends.google.GoogleOAuth2",
    "rest_framework_social_oauth2.backends.DjangoOAuth2",
    "django.c
```